config/locales/simple_form.en.yml

```diff
@@ -12,11 +12,17 @@ en:
       default_message: 'Please review the problems below:'
     labels:
       user:
+        full_name: Full name
```

Comment:
Do we really need setup label name here for custom model? We could do it here https://github.com/fs/rails-base/blob/master/config/locales/models/user.en.yml